service/openapi.yaml

```diff
@@ -279,3 +314,40 @@ components:
     errorDetails:
       type: string
       description: An error description for logging or other needs
+    metadataResponse:
+      description: OFREP metadata response
+      properties:
+        name:
+          type: string
+          description: name of the flag management system
+          examples:
+            - flagd
+            - go-feature-flag
+        features:
```

Comment:
As @jonathannorris said

> Not a fan of using features here, as most providers call themselves Feature Management Platforms now, and many systems have the concept of features as a top-level objects that will cause confusion here. I'd stick to calling this configuration / options something in that line.

I would also prefer something like configuration here.
Feature is cool for this case but there might be an option in the future that is not directly bound to something you would call a feature. But as I am struggling to come up with an example it might also be fine to go with features.

Reply:
"capabilities" is a good fallback when you want "features" in your feature management. Assuming that they are features/capabilities "I support the polling capability."

Reply:
Thats also nice, I am still just not sure if there will be configurations that are not `capabilities` @kinyoklion.
But as said that is just feeling I am still struggling to come up with an example.

Reply:
I like capabilities because this is really the goal.

And if we have other configurations (i don't think of any use case now) we can always add another key in the API.